src/utils/wallet.js

```diff
@@ -457,6 +457,18 @@ const wallet = {
   turnNotificationOff() {
     hathorLib.storage.setItem('wallet:notification', false);
   },
+
+  /**
+   * Get the server URL that the wallet will connect. Null if no URL is set
+   *
+   * @return {string} Server URL
+   *
+   * @memberof Wallet
+   * @inner
+   */
+  getServerURL() {
```

Comment:
Are you using this?

Reply:
You're right, this isn't being used. In fact, it's not even doing what it should be.

Removed it.